web/app/controllers/drivingSchoolCtrl.js

```diff
@@ -0,0 +1,49 @@
+var DrivingSchool = require("../models/DrivingSchool");
+var _ = require("underscore");
+var of = require("../libs/objectFunctions");
+
+// TODO: Check if Address, State, and Zip are valid (maybe some regex or external lib)
+exports.createSchool = function(info, callback, error) {
+	var requiredItems = ["addressLine1", "state", "zip"];
+	var missingItems = of.MissingProperties(info, requiredItems);
```

Comment:
I can see this MissingProperties function being useful for other areas in our code. I'll see if I can use this as well!